pkg/controller/kataconfig/openshift_reconciler.go

```diff
@@ -405,22 +412,61 @@ func (r *ReconcileKataConfigOpenShift) listKataPods() error {
 	return nil
 }
 
+func kataOcExists(r *ReconcileKataConfigOpenShift) (bool, error) {
+	kataOcMcp := &mcfgv1.MachineConfigPool{}
+	err := r.client.Get(context.TODO(), types.NamespacedName{Name: "kata-oc"}, kataOcMcp)
+	if err != nil && errors.IsNotFound(err) {
+		r.kataLogger.Info("No kata-oc machine config pool found!")
```

Comment:
I haven't checked, but I was wondering does this logger not have `Error` or do we must use `Info` to convey the error message?

Reply:
It does have Error, I changed it here and in a few other places added in this PR. Thanks!